.github/workflows/tests.yml

```diff
@@ -39,23 +39,15 @@ jobs:
         run: composer update --optimize-autoloader --no-progress --ansi --no-interaction
       - name: Build
         run: ./bin/unipoints build
+      - name: Check built assets
+        if: startsWith(matrix.os, 'windows') == false
+        run: |
+          if ! git diff --exit-code --name-status; then
+            printf '\n\nPLEASE RUN\n\nbin/unipoints build\n'
```

Comment:
```bash
echo "::error file=$(git diff --exit-code --name-status | head -n 1)::PLEASE RUN bin/unipoints build"
```
that creates a check annotation which is fancy 💄

Reply:
![image](https://github.com/mlocati/unipoints/assets/952007/37b170bb-7c3f-4c6e-84ad-7d6847460257)

Reply:
I've added https://github.com/mlocati/unipoints/pull/25/commits/8b94d64fcbf814b05934a8182c197b19237ecb0c to this PR